dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/query/DefaultQueryServiceTest.java

```diff
@@ -96,7 +95,7 @@ private List<OrganisationUnit> createOrgUnits(int size) {
 
     List<OrganisationUnit> result = new ArrayList<>();
     for (int i = 0; i < size; i++) {
-      result.add(createOrganisationUnit(RandomStringUtils.randomAlphabetic(1)));
+      result.add(createOrganisationUnit((char) (i + 65)));
```

Comment:
You can write `i + 'A'` for more clarity

Reply:
Thanks.